### PR TITLE
A few more mispellings

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
See https://github.com/Blosc/c-blosc2/pull/366.

### CODE REVIEW

**Review by peternewman — CHANGES_REQUESTED** [review]
Can you add `bumber` to bumped and I guess bummer. Maybe `bumby` too

**Comment by DimitriPapadopoulos** [comment]
And probably `bombed`.

**Comment by DimitriPapadopoulos** [comment]
Note than `bumb` is slang:
https://www.urbandictionary.com/define.php?term=bumb

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4487,9 +4488,16 @@ bulletted->bulleted
 bult->built
 bult-in->built-in
 bultin->builtin
-bumb->bump
+bumb->bump, bomb,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:4491]: Perhaps suggest bum too, which is what I'd expect for the homeless person or the bottom.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4487,9 +4488,16 @@ bulletted->bulleted
...
+bumbed->bumped, bombed,
+bumber->bumper, bomber,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:4493]:
```suggestion
bumber->bumper, bomber, bummer,
```

**Review by peternewman — CHANGES_REQUESTED** [review]
A few more sorry

**Review by peternewman — APPROVED** [review]

**Pull request merged**